benchmarks/distributed/lambda_runtimes/README.md

```diff
@@ -0,0 +1,15 @@
+Pull `python_resources` from `s3://tuplex-test-rahuly/python_resources.zip` (or copy from `../../../tuplex/awslambda/python38_resources.zip`)
```

Comment:
Something here is confusing. The zip refers to python3.8, yet we use python3.7 resources? 

Maybe clarifying what version we use and for what reason in the README might be a good idea.